rootfs-configs.yaml

```diff
@@ -110,7 +110,6 @@ rootfs_configs:
       - libext2fs2
     script: "scripts/buster-cros-ec-tests.sh"
     test_overlay: ""
-
```

Comment:
Please don't remove blank lines unnecessarily.  It causes noise in the diffs which can lead to git conflicts, and also blank lines help with readability of the file.